installer/x86_64/install.sh

```diff
@@ -446,7 +446,11 @@ fi
 # Decompress the file for the file system directly to the partition
 unzip -o $ONIE_INSTALLER_PAYLOAD -x "$FILESYSTEM_DOCKERFS" -d $demo_mnt/$image_dir
 
-TAR_EXTRA_OPTION="--numeric-owner"
+if [ "$install_env" != "build" ]; then
```

Comment:
what's this condition used for?

Reply:
build is the special environment.

Reply:
I think qi has not tested in the build environment.